src/app/pages/storage/components/unused-resources/unused-resources.component.ts

```diff
@@ -51,6 +63,16 @@ export class UnusedResourcesComponent implements OnInit {
       });
   }
 
+  protected addUnusedDisksToStorage(): void {
+    this.matDialog.open(ManageUnusedDiskDialogComponent, {
```

Comment:
It's weird to click on the button in one of the cards and to see both types of disks. I think `ManageUnusedDiskDialogComponent` should be reworked to just show disks that have been passed into it from the outside.